File: fizz-buzz.py
def fizz_buzz(x):
    if (x % 3 == 0) and (x % 5 == 0):
        return "FizzBuzz"
    elif x % 3 == 0:
        return "Fizz" if x % 5 != 0 else ""
    elif x % 5 == 0:
        return "Buzz"
    else:
        return str(x)

File: fizz-buzz.cpp
```c++
#include <iostream>
#include <string>

std::string fizzBuzz(int x) {
    if (x % 15 == 0)
        return "FizzBuzz";
    else if (x % 3 == 0)
        return x % 5 == 0 ? "FizzBuzz" : "Fizz";
    else if (x % 5 == 0)
        return "Buzz";
    else
        return std::to_string(x).c_str();
}

int main() {
    for(int i = 1; i <= 100; ++i) {
        std::cout << fizzBuzz(i) << std::endl;
    }
    return 0;
}